opencog/attentionbank/scm/attention-bank.scm

```diff
@@ -5,13 +5,10 @@
 
 (define-module (opencog attention-bank))
 
-; oc-config contains incorrect (insane) paths.
-; (use-modules (opencog) (opencog oc-config))
+; attention-config contains incorrect (insane) paths.
```

Comment:
this comment is probably no longer true, as it would seem that the point of this pull req is to fix the bug that lead to this comment ...

Reply:
I'm going to merge and then remove this comment by hand...